docs/reference/concepts/hyperscaling.md

```diff
@@ -1,152 +1,246 @@
+## Scaling Blockchains
 
-:::tip What is Scaling?
-Scaling = processing more transactions without degradation of security and decentralization.
-:::
+Today, Ethereum processes around 12 transactions per second. To support a future in which the world's financial activity is on-chain, we need to be able to process millions of transactions per second. However, blockchains are hard to scale due to their decentralized nature. The holy grail has always been horizontal scalability. Similar to the internet, where different servers host different websites, multiple distinct chains could host different applications in parallel. There have been many experiments attempting horizontal scaling, such as Polkadot, Cosmos, Near, and even the now-outdated Eth 2.0 vision, all of which have multiple shards or multiple chains. All of these solutions take care of interoperability, as the shards and bridges are designed together, as both are integral to the ecosystem. Unfortunately, the shards could never fully trust each other in any of these solutions.
 
-Scalability can be improved in a number of ways, from increasing the efficiency of the verifier to introducing probabilistic trust assumptions (what optimistic rollups do). Most of these approaches offer a linear scalability boost, but one method stands out: succinct zero-knowledge proofs (ZKPs). They always incur roughly O(1) verification costs regardless of the number of transactions processed. This means that ZKP-based scaling — i.e. [validiums and (under certain conditions) zkRollups](https://twitter.com/vitalikbuterin/status/1267455602764251138) — can be hyper-scalable:
+<div  align="center">
 
-:::tip What is Hyperscaling?
+![ ](../../assets/images/hyperscalingCosmos.png)
 
-Hyperscaling = processing infinitely many transactions without degradation of security and decentralization.
+The Cosmos ecosystem. Unfortunately, the bridges have trust requirements, the ecosystem ultimately does not scale. 
 
-:::
+</div>
 
-In other words, hyperscalability means breaking out of the blockchain trilemma:
 
-![Hyper-scalability!](../../assets/images/hyperscaling1.png "Hyper-scalability")
+In parallel, the Ethereum community was solving the trust problem by verifying off-chain computations on-chain, inventing Plasma, optimistic rollups, and other solutions. It is now clear that zero-knowledge proofs are the future, as they provide cryptographic security. Ethereum will be accessible to everybody. The [DA layer](https://ethereum.org/en/roadmap/danksharding/) and ZK Rollups together will provide a secure and scalable ecosystem. But how will the rollups interoperate? Now that we have figured out the building blocks, it is time to review the system as a coherent whole again.
```

Comment:
I suggest shortening this introduction to something as follows and remove the cosmos image:


Ethereum, currently processing around 12 transactions per second, must scale to handle millions of transactions for a future of global on-chain financial activities. Achieving this in a decentralized structure poses significant challenges. Various solutions, including Polkadot, Cosmos, Near, and Eth 2.0, have been explored with multi-chain or shard architectures, but full trust remains an issue. The widely accepted solution for these challenges lies in zero-knowledge proofs, offering cryptographic security. Combined with the DA layer and ZK Rollups, these can create a secure, scalable ecosystem, making Ethereum accessible to all.